Detectors/EMCAL/base/include/EMCALBase/Point.h

```diff
@@ -83,10 +90,11 @@ namespace AliceO2 {
       void PrintStream(std::ostream &stream) const;
       
     private:
-      Int_t         mShunt;             ///< Shunt (check if needed)
-      Int_t         mPrimary;           ///< Primary particles at the origin of the hit
-      Int_t         mParent;            ///< Parent particle that entered the EMCAL
-      Double32_t    mInitialEnergy;     ///< Energy of the parent particle that entered the EMCAL
+      Vector3D<float>   mPvector;           ///< Momentum Vector
+      Int_t             mShunt;             ///< Shunt (check if needed)
```

Comment:
Fully agree to abandon pointers where possible. I will change to Point3D and Vector3D as arguments.